lib/src/text-input/TextInput.test.js

```diff
@@ -643,6 +645,54 @@ describe("TextInput component synchronous autosuggest tests", () => {
     expect(input.value).toBe("");
     expect(queryByRole("listbox")).toBeFalsy();
   });
+
+  test("Autosuggest escapes special characters", () => {
+    const onChange = jest.fn();
+    const { getAllByText, getByText, getByRole } = render(
+      <DxcTextInput label="Autocomplete Countries" suggestions={special_characters} onChange={onChange} />
+    );
+    const input = getByRole("combobox");
+    fireEvent.focus(input);
+    const list = getByRole("listbox");
+    fireEvent.change(input, { target: { value: "/" } });
+    expect(list).toBeTruthy();
+    expect(getAllByText("/").length).toBe(1);
+    expect(getByText("/")).toBeTruthy();
```

Comment:
Isn't this redundant to the previous expect?

Reply:
It is just double check, but yes, it is a bit redundant.

Reply:
Done